email_account_validity/_store.py

```diff
@@ -275,13 +327,21 @@ async def get_user_from_renewal_token(
                 * An optional int representing the timestamp of when the user renewed
                     their account timestamp as milliseconds since the epoch. None if the
                     account has not been renewed using the current token yet.
+
+        Raises:
+            SynapseError(404): The token could not be found (or does not belong to the
```

Comment:
We can be more specific:

```suggestion
            StoreError(404): The token could not be found (or does not belong to the
```